swarm/storage/chunker.go

```diff
@@ -510,13 +506,13 @@ func (s *LazyChunkReader) Seek(offset int64, whence int) (int64, error) {
 	case 1:
 		offset += s.off
 	case 2:
-		if s.chunk == nil { //seek from the end requires rootchunk for size. call Size first
+		if s.chunkData == nil { //seek from the end requires rootchunk for size. call Size first
 			_, err := s.Size(nil)
 			if err != nil {
 				return 0, fmt.Errorf("can't get size: %v", err)
 			}
 		}
-		offset += s.chunk.Size
+		offset += s.chunkData.Size()
```

Comment:
@nolash I think you have to specify a negative offset in case of SeekEnd whence, so no. For sure I did not change how this works, just modified the chunk to chunkData.